Orange/canvas/report/owreport.py

```diff
@@ -191,6 +191,10 @@ def _remove_item(self, row):
         self.table_model.removeRow(row)
         self._build_html()
 
+    def _clear(self):
```

Comment:
I don't see this function used anywhere. Is there a good reason for introducing dead code here?